docs/reference/troubleshooting/data/increase-cluster-shard-limit.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[increase-cluster-shard-limit]]
+== Maximum number of shards per node has been reached
```

Comment:
```suggestion
== Total number of shards per node has been reached
```
Please adjust this to `total` to differentiate from `max_shards_per_node` setting